docs/en/observability/synthetics-recorder.asciidoc

```diff
@@ -0,0 +1,84 @@
+// Content type: Reference(ish)
+//
+// Purpose: Provide complete overview of the Synthetics Recorder UI.
+//
+// Structure: Reflects the pieces of the UI. Also somewhat sequential.
+//
+// Audience assumptions:
+// - The reader may or may not be familiar with Elastic Synthetics already.
+// - The reader might have come here from "Write a synthetic test" or they might have
+// arrived here from a search without larger no context.
+// - The reader might be using Project monitors, Heartbeat, or Elastic Agent.
+// 
+// What's not included:
+// - Detailed definitions of journey, step, etc.
+// - Detailed instructions on how to create a monitor. Point to instructional guide instead.
+
+[[synthetics-recorder]]
+= Synthetics Recorder
+
+You can use the Synthetics Recorder to <<synthetics-create-test>> by interacting with a web page and exporting journey code that reflects all the actions you took.
+
+[discrete]
+== Set up
+
+// Rough requirements to cover
+Requirements:
+* **Elastic Synthetics Recorder**. Download and install the Synthetics Recorder from the release page.
+* **Chrome**. Download and install the Chrome browser. This will be needed to ...
+
+[discrete]
+== Record a journey
+
+// Rough steps to cover
+Enter a starting URL in the search box.
+// README: After starting up the application, you may input a URL. This URL will be the starting point of the journey script Elastic Synthetics will create.
+
+Click **Start** and a Chromium window will open on the page you specified. 
+Navigate the page to record actions.
+// README: Once you start a journey, Elastic Synthetics will record actions based on your interaction with the browser window. This includes clicking on text, navigation, focusing on inputs like buttons and text fields, and more. As you complete your journey, you will see the actions you are generating populate in the Script Recorder's window.
+
+Click pause to ...
+// README: The Script Recorder also includes a Pause feature. When you pause the recording session, you may click around the browser window without recording any actions. Un-pause the Recorder to continue recording journey actions.
+
+Click **Stop** to stop recording.
+
+[discrete]
+== Edit a journey
+
+// Rough topics to cover
+Name step(s)
+
+Edit action
+
+Add an assertion
+// README: You can also add assertions to your journey. Use these to make determinations about the state of the page you are testing. Assertions can include checks for things like the visibility of an element, or the contents of a text field.
+
+Delete an action
+
+Split into multiple steps
+
+[discrete]
+== Test the journey
+
+// README: At any point during or after the recording process concludes, you may test your script. When you click the Test button, Elastic Synthetics will run the journey you have defined. After the test concludes, the Recorder will display results on a per-step basis.
```

Comment:
```suggestion
// README: At any point during or after the recording process concludes, you may test your script. When you click the Test button, Elastic Synthetics will run the journey you have defined. As the test runs, the Recorder will display results on a per-step basis.
```

Thanks to some great work by @kyungeunni the recorder will actually respond in real time as the test is running with step-level updates.